regions/io/ds9/read.py

```diff
@@ -5,42 +5,38 @@
 import re
```

Comment:
Apart from changing things into `io.core`, are the other updates in `io.ds9` required in this pull request?

Reply:
Some of them are required, there have been some minor changes in the shape class. There are also some doc fixes which could have gone in a separate PR(My bad!)